pkg/cmd/codespace/ssh.go

```diff
@@ -144,6 +144,33 @@ func newSSHCmd(app *App) *cobra.Command {
 	return sshCmd
 }
 
+type ReadWriteHalfCloser interface {
```

Comment:
This was previously in the `liveshare` package so it was moved here as this was the only place it was used.

Reply:
Definitely! Updated.